TommyPROM/TommyPROM.ino

```diff
@@ -722,16 +728,30 @@ void loop()
         break;
 
     case CMD_DUMP:
-        dumpBlock(start, end);
+        start = dumpBlock(start, start + if_unspec(end, 0xff));
```

Comment:
I think this should be 
if_unspec(end, start+0xff)
otherwise d 100 1ff would dump from 100 to 2ff instead of 100 to 1ff
Am I reading that right?

Reply:
Yes that's right; will fix.